<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiscreteRangeGUI</class>
 <widget class="QWidget" name="DiscreteRangeGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>30</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>172</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="rangeLabel">
      <property name="text">
       <string>Label</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="rangeCombo"/>
    </item>
    <item>
     <widget class="QLabel" name="rangeUnits">
      <property name="text">
       <string>Unit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
